server/traces/otel_converter.go

```diff
@@ -0,0 +1,102 @@
+package traces
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/kubeshop/tracetest/openapi"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func FromOtel(trace openapi.ApiV3SpansResponseChunk) (Trace, error) {
+	flattenSpans := make([]openapi.V1Span, 0)
+	for _, resource := range trace.ResourceSpans {
+		for _, librarySpans := range resource.InstrumentationLibrarySpans {
```

Comment:
Remember that there are some attributes at the resource level as well. Things like `service.name` are there.

Reply:
Good call. I think only the span name is at the resource top level. I'm almost sure service.name is inside the `attributes` array.